new-examples/raw-templates/raw_templates.bal

```diff
@@ -1 +1,33 @@
+import ballerina/io;
+import ballerinax/java.jdbc;
+import ballerina/sql;
 
+jdbc:Client dbClient = check new (url = "jdbc:h2:file:./master/orderdb",
```

Comment:
@lasinicl We didn't use other std libs so far. Shall come up with an example without an std lib. @sameerajayasoma wdyt ?